maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/service/ProceedingsServiceTest.java

```diff
@@ -71,4 +78,60 @@ void givenWorkflowRequest_whenUpdateCrownCourtIsInvoked_thenApiServiceIsCalledAn
                                                           any(ApplicationDTO.class)
                 );
     }
+
+    @Test
+    void givenAAppealToCC_whenCanInvokeMsgRepDecisionIsInvoked_thenFalseIsReturned() {
+        ApplicationDTO applicationDTO = TestModelDataBuilder.getApplicationDTO();
+        applicationDTO.getCaseDetailsDTO().setCaseType(CaseType.APPEAL_CC.getCaseType());
+        assertThat(proceedingsService.canInvokeMagsRepDecision(applicationDTO)).isFalse();
```

Comment:
See previous comments, this method should be private. Instead, the tests should call `determineMagsRepDecision`, which represents the public interface. I'd suggest using parameterised tests to make this easier.